src/components/ui/Button/index.js

```diff
@@ -138,6 +139,11 @@ export const Button = (props) => {
           outline: none;
         }
 
+        ${noMargin &&
+          css`
+            margin: 0;
```

Comment:
Why _override_ the value here? Wouldn't it make more sense to only output `margin: 0 ${toUnits(theme.spacing.margin.xxSmall)} ${toUnits(theme.spacing.margin.xSmall)};` from above is `noMargin === false`?

Unless the aim here is to reset browser styles, this seems like an extra step

Reply:
To account for browser defaults, we definitely need to pass margin: 0 values to the components, especially stuff like headings and paragraphs. We could avoid outputting the default margins and then resetting them at the expense of making the component code a bit more complex:
```

        ${noMargin &&
          css`
            margin: 0 ${toUnits(theme.spacing.margin.xxSmall)}
              ${toUnits(theme.spacing.margin.xSmall)};
          `}

        ${!noMargin &&
          css`
            margin: 0;
          `}
```
Should I refactor it?

Reply:
Nah, I don't think that's worth it, and technically it'll add another class to the output (that's how `emotion` works under-the-hood).

If the `margin: 0` is required, I'm fine to leave it as-is!

Reply:
I've gone ahead and changed the logic here so the first value we pass is a margin of 0, but if noMargin is false (the default behavior) we override that zero-margin with one of our own. Let me know how that works!